src/sentry/api/endpoints/auth_index.py

```diff
@@ -151,15 +184,11 @@ def put(self, request: Request):
                 status=status.HTTP_403_FORBIDDEN,
             )
 
-        if request.user.is_superuser and not is_active_superuser(request) and Superuser.org_id:
-            # if a superuser hitting this endpoint is not active, they are most likely
+        if from_superuser_modal:
```

Comment:
what if a non-superuser accesses the superuser modal? should you still check that the user is superuser here?

Reply:
from the frontend it shouldn't possible to open the superuser if the user isn't a non-superuser, but the line right under this checks if the user is a superuser and if they're not an active superuser